canvas_modules/common-canvas/__tests__/test_resources/paramDefs/code_paramDef.json

```diff
@@ -0,0 +1,71 @@
+{
+  "titleDefinition": {
+    "title": "Code Control Tests"
+  },
+  "current_parameters": {
+    "code": "Age >= 55",
+    "code_rows": ""
+  },
+  "parameters": [
+    {
+      "id": "code",
+      "type": "string"
+    },
+    {
+      "id": "code_rows",
+      "type": "string"
+    }
+  ],
+  "uihints": {
+    "id": "code.test",
+    "label": {
+      "default": "Code Test"
+    },
+    "editor_size": "medium",
+    "description": {
+      "default": "Test code controls"
+    },
+    "parameter_info": [
+      {
+        "parameter_ref": "code",
+        "language": "text/x-python",
+        "label": {
+          "default": "Code"
+        },
+        "description": {
+          "default": "Enter Python code"
+        },
+        "text_before": {
+          "default": "Press ctrl-space to autocomplete, enter a '?' for validate to fail."
```

Comment:
We should remove the second half of this sentence since validation doesn't exist for code controls